static/sass/components/_email-subscribe.scss

```diff
@@ -2,24 +2,20 @@
 // ==============================
 
 .odd-input-group {
-  [data-region='footer'] & {
-    align-items: flex-end;
-    display: flex;
+  align-items: flex-end;
```

Comment:
When this is moved out from the footer region, the input and button are inline and not the same size. I thought I would make the footer inline, but the body stacked (like the contact form) so I don't have to fidget with the line-height/padding/height of the button and inputs matching.

Reply:
I thought inline looked good both places. See https://github.com/oddbird/oddsite/pull/316 for my attempt at cleaning this up with padding adjustments.